packages/bruno-app/src/components/CollectionSettings/Auth/index.js

```diff
@@ -6,6 +6,7 @@ import AwsV4Auth from './AwsV4Auth';
 import BearerAuth from './BearerAuth';
 import BasicAuth from './BasicAuth';
 import DigestAuth from './DigestAuth';
+import WsseAuth from './WsseAuth';
```

Comment:
Please add a new case in the function `getAuthView` for `wssewauth` and return `<WsseAuth collection={collection} />`

<img width="628" alt="Screenshot 2024-09-03 at 4 35 38 PM" src="https://github.com/user-attachments/assets/512dc5db-5e3c-475d-a3f0-23a46786b134">

Reply:
Assuming here you meant 'wsseauth' instead of 'wssewauth'. Updated.